src/examples/example.ts

```diff
@@ -1,10 +1,20 @@
 import * as THREE from 'three';
 import ComboControls from '../';
+declare var require: any;
```

Comment:
This shouldn't be necessary or am I mistaken?

Reply:
require is not defined in typescript.
https://stackoverflow.com/questions/31173738/typescript-getting-error-ts2304-cannot-find-name-require

Reply:
Could you rather do the TypeScript 2.x solution?